ambari-metrics-timelineservice/src/test/java/org/apache/ambari/metrics/webapp/TestTimelineWebServices.java

```diff
@@ -75,28 +73,28 @@ protected Injector getInjector() {
 
   public TestTimelineWebServices() {
     super(new WebAppDescriptor.Builder(
-      "org.apache.ambari.metrics.webapp")
-      .contextListenerClass(GuiceServletConfig.class)
-      .filterClass(com.google.inject.servlet.GuiceFilter.class)
-      .contextPath("jersey-guice-filter")
-      .servletPath("/")
-      .clientConfig(new DefaultClientConfig(YarnJacksonJaxbJsonProvider.class))
-      .build());
+            "org.apache.ambari.metrics.webapp")
+            .contextListenerClass(GuiceServletConfig.class)
+            .filterClass(com.google.inject.servlet.GuiceFilter.class)
+            .contextPath("jersey-guice-filter")
+            .servletPath("/")
+            .clientConfig(new DefaultClientConfig(YarnJacksonJaxbJsonProvider.class))
+            .build());
   }
 
   @Test
   public void testAbout() throws Exception {
     WebResource r = resource();
     ClientResponse response = r.path("ws").path("v1").path("timeline")
-      .accept(MediaType.APPLICATION_JSON)
-      .get(ClientResponse.class);
+            .accept(MediaType.APPLICATION_JSON)
```

Comment:
I think you're using an indent width of 4 instead of 2.

Reply:
No, I checked, I am using proper indentation, seems like the continued lines are being indented by +2